### DISCUSSION TITLE
Add a command line argument to rename/move config folder

### DISCUSSION THREAD

**Discussion by ghost** [PurpurMC/Purpur#1005] [__Completed__] [answered]
Same as PaperMC/Paper#7948 and pufferfish-gg/Pufferfish#35

Add a `--purpur-settings` command-line argument to the startup flags so we are able to specify where purpur.yml is saved. For example `--paper-settings-directory "settings/"` saves all the paper configs to the settings/paper.yml folder within the server directory.

So it could be for example `--purpur-settings "settings/purpur.yml"`

I know you are currently working on implementing the new config system so not sure if this was thought of for implementation but it would be nice.

**Comment by BillyGalbreath** [comment, marked as answer]
https://github.com/PurpurMC/Purpur/blob/ver/1.19/patches/server/0004-Purpur-config-files.patch#L74-L82
